docs/docs/fargate-tutorials/django.md

```diff
@@ -3,7 +3,7 @@ sidebar_position: 3
 slug: '/django'
 ---
 
-# IaSQL on Django (Python)
+# Deploy AWS ECS Fargate using IaSQL on Django (Python)
```

Comment:
what about:
```suggestion
# Deploy a containerized application on AWS Fargate using IaSQL and Django (Python)
```

Reply:
this title will show in the docs sidebar so that will get hidden, but a compromise is `Deploy container to ECS Fargate using IaSQL on Django (Python)`

Reply:
better?